docs/content/tutorials/nginx-ingress-linkerd.md

```diff
@@ -0,0 +1,128 @@
+---
+title: NGINX Ingress Controller and Linkerd
+description: |
+  Use NGINX Ingress Controller with Linkerd
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1232"
+---
+
+# NGINX Ingress controller with Linkerd
+
+This document will walk you through how to install NGINX Ingress controller with Linkerd. Linkerd works with both NGINX Ingress controller open source and NGINX Ingress controller plus version.
+
+We are going to install the Linkderd sidecar proxy with NGINX Ingress controller.
+
+We need to install the `Linkerd` control plane utility.
+Linkerd has a great document on how to install the linkerd control plane utility on their website.
+You can find the install steps on `Linkerd` website: [Linkerd Service mesh](https://linkerd.io/2.13/getting-started/)
```

Comment:
Instead of saying "We need to install" have this section the a stepped section. 
For example
```
## Step 1 - Installing Linkerd
Follow steps outlined on the `Linkerd` [getting started](https://linkerd.io/2.13/getting-started/) page to installing the `Linkerd` control plane utility
```